docs/analyzing-dependencies.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Analyzing Dependencies
+category: 61e72e3a50a88e001a92ee5d
+---
+
+The Phylum CLI natively supports processing the lock/requirements files for several ecosystems, namely:
+* NPM
+* Ruby Gems
+* PyPI
+
+After setting up a Phylum [project](doc:projects) , you can begin analysis by running:
```

Comment:
Interesting...does the `doc:projects` form of relative linking work for ReadMe.com? Using [relative links](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links) should work for both sources (GitHub and ReadMe.com). There are three (3) instances of `doc:<something>` in this review.

Reply:
It is a relative link mechanism for ReadMe.com, I have removed it in favor of a full link. I only noticed this one link, not three (3), so let me know if I missed something.